assets/wizards/popups/components/settings-modal/index.js

```diff
@@ -155,6 +157,44 @@ const PromptSettingsModal = ( { prompt, disabled, onClose, segments, updatePopup
 							'newspack'
 						) }
 					/>
+					<Button isLink onClick={ () => setShowAdvanced( ! showAdvanced ) }>
+						{ sprintf(
+							// Translators: whether to show or hide advanced settings fields.
+							__( '%s Advanced Settings', 'newspack_popups_taxonomy' ),
+							showAdvanced ? __( 'Hide', 'newspack' ) : __( 'Show', 'newspack' )
+						) }
+					</Button>
+					{ showAdvanced && (
+						<>
+							<CategoryAutocomplete
+								label={ __( 'Category Exclusions', 'newspack ' ) }
+								disabled={ disabled }
```

Comment:
Disabling the component interferes with user input—since it's kind of a multi-select component, we should allow users to continue inputting text in the field while it's fetching suggestions. 980a53f implements a visual loading state so that the user at least knows something is happening during that time, though.